documentation/src/main/asciidoc/userguide/appendices/Configurations.adoc

```diff
@@ -54,6 +54,14 @@ Note that for backwards compatibility, if a https://docs.jboss.org/hibernate/orm
 |`hibernate.connection.isolation` | `REPEATABLE_READ` or
 `Connection.TRANSACTION_REPEATABLE_READ` | Names the JDBC connection transaction isolation level.
 |`hibernate.connection.autocommit` | `true` or `false` (default value) | Names the JDBC connection autocommit mode.
```

Comment:
Strictly speaking this really is a transaction-related setting as opposed to connection-related - although to be fair, so is `hibernate.connection.autocommit`.  Anyway, because it is transaction-related I wonder if this new setting should be `hibernate.transaction.skip_autocommit_check` .

Also, I'm not sure the `skip_autocommit_check` portion is even accurate.  You are not skipping a check so much as skipping disabling autocommit (on the premise that it is already so).